excise/tasks.py

```diff
@@ -0,0 +1,101 @@
+import json
+import logging
+
+import opentracing
+import opentracing.tags
+
+from saleor.celeryconf import app
+from saleor.core.taxes import TaxError
+from saleor.order.events import external_notification_event
+from saleor.order.models import Order
+from .utils import (
+    AvataxConfiguration,
+    api_post_request,
+    api_commit_transaction,
+    get_metadata_key,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@app.task(
+    autoretry_for=(TaxError,),
+    retry_backoff=60,
+    retry_kwargs={"max_retries": 5},
+)
+def api_post_request_task(
+    transaction_url, data, config, order_id, commit_url=None
+):
+    config = AvataxConfiguration(**config)
+    order = Order.objects.filter(id=order_id).first()
+    if not order:
+        msg = (
+            f"Unable to send the order {order_id} to Avatax Excise. "
+            "Order doesn't exist."
+        )
+        logger.error(msg)
```

Comment:
Maybe? This will be better handled by tools like sentry.
```suggestion
        msg = (
            "Unable to send the order %s to Avatax Excise. "
            "Order doesn't exist."
        )
        logger.error(msg, order_id)
```

Reply:
Thanks! Added a fix here: https://github.com/saleor/saleor-plugin-avatax-excise/pull/7/commits/ce95dc0c8af772bf2e69d5262942480131d82c78